options/locale/locale_en-US.ini

```diff
@@ -732,6 +732,32 @@ issues.add_time_sum_to_small = No time was entered
 issues.cancel_tracking = Cancel
 issues.cancel_tracking_history = `cancelled time tracking %s`
 issues.time_spent_total = Total time spent
+issues.dependency.title = Dependencies
+issues.dependency.issue_no_dependencies = This issue currently doesn't have any dependencies.
+issues.dependency.pr_no_dependencies = This pull request currently doesn't have any dependencies.
+issues.dependency.add = Add a new dependency...
+issues.dependency.cancel = Cancel
+issues.dependency.remove = Remove
+issues.dependency.issue_number = Issuenumber
+issues.dependency.added_dependency = `<a href="%[1]s">%[2]s</a> added a new dependency %[3]s`
+issues.dependency.removed_dependency = `<a href="%[1]s">%[2]s</a> removed a dependency %[3]s`
+issues.dependency.issue_closing_blockedby = Closing this pull request is blocked by the following issues.
+issues.dependency.pr_closing_blockedby = Closing this issue is blocked by the following issues.
+issues.dependency.issue_close_blocks = This issue blocks closing of the following issues.
+issues.dependency.pr_close_blocks = This pull request blocks closing of the following issues.
+issues.dependency.issue_close_blocked = You need to close all issues blocking this issue before you can close it!
+issues.dependency.pr_close_blocked = You need to close all issues blocking this pull request before you can merge it!
+issues.dependency.blocks_short = Blocks
+issues.dependency.blocked_by_short = Blocked by
```

Comment:
The jargon used on sidebar seems to me inconsistent. I'd rather have `Depends on` for "blocked by" and `Depend on %s` for "Block". This way we get an imo clearer version:

```
Depends on: #1, #2, #3
Depend on #4: #5
```

Reply:
I don't think "Depend on" would be clear enough. I'm okay with "Depends on" though.

Reply:
Yeah, I agree, that's why I suggested the %s part, so that it would be more explicit that the current issue is the object.